apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/controller/OnboardingController.java

```diff
@@ -333,4 +333,15 @@ public Uni<Response> update(@PathParam(value = "onboardingId") String onboarding
     public Uni<Boolean> checkManager(OnboardingUserRequest onboardingUserRequest) {
         return onboardingService.checkManager(onboardingUserRequest);
     }
+
+    @Operation(summary = "Perform onboarding aggregation as /onboarding but completing the onboarding request to COMPLETED phase.")
```

Comment:
if we set timeout null, you could add some comments about that the operation will perform async